main.js

```diff
@@ -0,0 +1 @@
+alert("Hi");
```

Comment:
Alerts are often considered annoying. Change this to console.log

Reply:
I thought I deleted that! That was only in there to check that I had set up the js correctly for the next stage

Reply:
Yeah I figured. This is why we do PRs :)